starter-code/app.js

```diff
@@ -1,5 +1,60 @@
+'use strict';
+
 const express = require('express');
 const app = express();
-const Chuck  = require('chucknorris-io');
+const Chuck = require('chucknorris-io');
 const client = new Chuck();
+const expressLayouts = require('express-ejs-layouts');
+
+// app.get('/', (req, res, next) => {
+//   res.send('Hello');
+// });
+
+
+// -- setup
+app.use(express.static('public'));
+app.use(expressLayouts);
+app.set('layout', 'layouts/main-layout');
+app.set('views', __dirname + '/views');
+app.set('view engine', 'ejs');
+
+
+// -- routes
+
+app.get('/random', (req, res) => {
+  client.getRandomJoke()
+    .then((response) => {
```

Comment:
You can call this "response" (or whatever you want) but may be confusing because you have a variable called "res" in the route. Better call it "joke". That is what the API is giving you. Remember to change the variable's name in the line 28 too.